src/observables/CallbackObservable.ts

```diff
@@ -0,0 +1,101 @@
+import {Observable} from '../Observable';
+import {Subscriber} from '../Subscriber';
+import {Scheduler} from '../Scheduler';
+import {Subscription} from '../Subscription';
+import {immediate} from '../schedulers/immediate';
+import {tryCatch} from '../util/tryCatch';
+import {errorObject} from '../util/errorObject';
+
+export class CallbackObservable<T> extends Observable<T> {
```

Comment:
What about align class name to observable, such as `Observable.fromEvent` has implementation named `FromEventObservable`?